django/core/management/commands/loaddata.py

```diff
@@ -84,11 +84,20 @@ def loaddata(self, fixture_labels):
         }
         if has_bz2:
             self.compression_formats['bz2'] = (bz2.BZ2File, 'r')
-
+        # Django's test suite repeatedly tries to load initial_data fixture
+        # from apps that do not have any fixtures. Because disabling constraint
+        # checks can be expensive on some database (especially MSSQL), bail
+        # out early if no fixtures are found.
+        found_fixtures = False
+        for fixture_label in fixture_labels:
+            if len(self.find_fixtures(fixture_label)) > 0:
+                found_fixtures = True
+                break
+        if not found_fixtures:
+            return
```

Comment:
Small nitpick but since an empty `list` evaluates to `False` the `len(...) > 0` isn't required and makes the whole check less readable IMHO.